core/src/avm2/globals/flash/utils.rs

```diff
@@ -25,6 +25,66 @@ pub fn get_timer<'gc>(
         .into())
 }
 
+/// Implements `flash.utils.setInterval`
+pub fn set_interval<'gc>(
+    activation: &mut Activation<'_, 'gc, '_>,
+    _this: Option<Object<'gc>>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error> {
+    let (args, params) = args.split_at(2);
+    let callback = crate::timer::TimerCallback::Avm2Callback {
+        closure: args.get(0).unwrap().as_object().unwrap(),
+        params: params.to_vec(),
+    };
+    let interval = args.get(1).unwrap().coerce_to_number(activation)?;
+    Ok(Value::Integer(activation.context.timers.add_timer(
+        callback,
+        interval as i32,
+        false,
+    )))
+}
+
+/// Implements `flash.utils.clearInterval`
+pub fn clear_interval<'gc>(
+    activation: &mut Activation<'_, 'gc, '_>,
+    _this: Option<Object<'gc>>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error> {
+    let id = args.get(0).unwrap().coerce_to_number(activation)?;
```

Comment:
Should make sure we have an argument beforehand, or do handle not having an argument properly.